### DISCUSSION TITLE
The "Public Sponsor Achievement" that failed the transaction

### DISCUSSION THREAD

**Discussion by LianSheng197** [community/community#33837] [Other Feature Feedback, Questions, & Ideas]
Hello, there is a small problem with "Public Sponsor Achievement", I don't know how to report it to GitHub, so here it is.

"Public Sponsor Achievement" appears when we try to sponsor a project and the transaction fails.
In my case, I tried to sponsor Flarum on Aug 15, but due to some factors, I wasn't able to sponsor until just now. During this time, I got "Public Sponsor Achievement", the only difference is that the red box shows 0.  
![Selection_595](https://user-images.githubusercontent.com/18535495/191773466-8eb95a33-61f6-448b-980a-8d681c117801.png)

Personally, I don't think this initial achievement should have happened since the sponsorship transaction didn't hold.

Lastly, I'm not a native English speaker, so I'm sorry if my description confuses you.

**Comment by radar-blip** [comment]
I'm assuming this is fixed now. Maybe it just took a little time.
Your Public Sponsor badge now shows you are sponsoring 2 orginizations. 😃

**Reply by LianSheng197** [reply]
no, i mean the question before this
I didn't find this forum until after the transaction was successful
Otherwise I should have asked this question earlier
